runtime/tests/interpreter/interpreter_test.go

```diff
@@ -6615,6 +6615,8 @@ func TestInterpretContractAccountFieldUse(t *testing.T) {
 						return map[string]interpreter.Value{
 							"account": interpreter.NewAuthAccountValue(
 								addressValue,
+								func() interpreter.UInt64Value { return interpreter.UInt64Value(0) },
```

Comment:
Maybe refactor this to a function declaration, e.g. and replace the line here and in the other cases below:

```go
func returnZero() interpreter.UInt64Value {
	return interpreter.UInt64Value(0)
}
```

```suggestion
								returnZero,
```